### ISSUE TITLE
about iOS 12.5.7

### ISSUE THREAD

**Issue by iamnabilzaman** [pwned4ever/Th0r_Freya#43]
is there any way to jailbreak with freya on iPhone 5s running on iOS 12.5.7?

**Comment by pwned4ever** [comment]
Yes I have multiple devices include 5s on iOS 12.5.7 running freya. What seems to be your issue?

Sent from my iPhone

On Jun 21, 2023, at 1:24 PM, voidLinux ***@***.***> wrote:

﻿

is there any way to jailbreak with freya on iPhone 5s running on iOS 12.5.7?

**Comment by iamnabilzaman** [comment]
it's crashing automatically plus after crash enable freya cause sudden reboot

**Comment by iamnabilzaman** [comment]
[
![Screenshot (3)](https://github.com/pwned4ever/Th0r_Freya/assets/94600437/7c40d697-027e-405e-a271-22dd91dc8102)
](url)

**Comment by pwned4ever** [comment]
Try to restore rootfs settings section of the app.

Message ID: ***@***.***>

After that try start fresh with jailbreaking again. Record the process from another device if possible and show me all runs of freya. From removing to fresh installing and jailbreaking from fresh start

**Comment by iamnabilzaman** [comment]
[CrashLog.txt](https://github.com/pwned4ever/Th0r_Freya/files/11847264/CrashLog.txt)

**Comment by iamnabilzaman** [comment]
@pwned4ever here is the video
[Uploading WhatsApp Video 2023-06-24 at 00.20.39.zip…]()

**Comment by themaluxis** [comment]
Exaclty same issue on my side

**Comment by pwned4ever** [comment]
Same issue as? IPad Air? No visible Cydia icon?

Sent from my iPhone

On Oct 22, 2023, at 4:14 AM, Maxence ***@***.***> wrote:

﻿

Exaclty same issue on my side

**Comment by themaluxis** [comment]
Same as [iamnabilzaman](https://github.com/iamnabilzaman)
"it's crashing automatically plus after crash enable freya cause sudden reboot"
